### PR TITLE
Post-CR Changes on ECHIDNA (DO NOT MERGE THIS BEFORE CR PUBLICATION)

### PR DESCRIPTION
**Pull request by iherman**
With the CR publication, the usage of ECHIDNA changes (see [separate ECHIDNA page](https://github.com/w3c/echidna/wiki/CR-publication-workflow)):

1. Any editorial CR update can be made on a CR via ECHIDNA
2. A substantial CR update must require a Director's approval, and has to go through the staff contact and the W3C Webmaster

This PR makes the necessary changes on the ECHIDNA setup.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,4 +14,4 @@ script:
   - echo "ok"
  
 after_success:
-  - test $TRAVIS_PULL_REQUEST = false && curl "https://labs.w3.org/echidna/api/request" --data "url=$URL" --data "decision=$DECISION" --data "token=$TOKEN"
+  - test $TRAVIS_PULL_REQUEST = false && curl "https://labs.w3.org/echidna/api/request" --data "url=$URL" --data "decision=$DECISION" --data "token=$TOKEN" --data="editorial=true"
```
Comment (deniak) [inline .travis.yml:17]:
```suggestion
  - test $TRAVIS_PULL_REQUEST = false && curl "https://labs.w3.org/echidna/api/request" --data "url=$URL" --data "decision=$DECISION" --data "token=$TOKEN"
```

File: ECHIDNA [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # ECHIDNA configuration
-index.html?specStatus=WD&shortName=did-core respec
+index.html?specStatus=CR&shortName=did-core respec
```
Comment (deniak) [inline ECHIDNA:2]:
```suggestion
index.html?specStatus=CRD&shortName=did-core respec
```

**Review by deniak — CHANGES_REQUESTED** [review]
The documentation is actually no longer up-to-date. With process 2020, we now have a better way to differentiate the 2 types of CR (CR Snapshot and CR Draft). For updates that are not substantive, it should be a CR draft.

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Purely editorial, merging.

**Pull request merged**